### ISSUE TITLE
NVDA is not reading the Toast notification message when the button is clicked

### ISSUE THREAD

**Issue by bhavya974** [nvaccess/nvda#9062]
NVDA is not reading the toast notification message when the button is clicked.

**Steps to reproduce:**
 1. Click on the link given below.
     Link: [https://www.cssscript.com/demo/super-simple-javascript-message-toaster-toast-js/](url)
 2. Open it in Chrome Browser.
 3. Start the NVDA.
 4. Click on the button, a toast message is displayed.

**Actual behavior:**
 -->A toast notification message pops up, but the NVDA doesn't read the toast message.

**Expected behavior:**
--> Screen Reader should read all the contents and controls of the page properly.The NVDA should read the toast message when it pops up.

**System configuration**
**NVDA installed/portable/running from source: Installed**

**NVDA version: 2018.2.1**

**Windows version: 10.0.17763 Build 17763**

**Name and version of other software in use when reproducing the issue:**

**Other information about your system:**
**Chrome**- Version 70.0.3538.110 (Official Build) (64-bit)

**Other questions**
**Does the issue still occur after restarting your PC?**
Yes, it does.

**Have you tried any other versions of NVDA?**
No, I didn't.

**Comment by Adriani90** [comment]
According to this source toasts need to have a aria alert role or aria-live role in order to be automatically announced by the screen reader.
https://sheribyrnehaber.medium.com/designing-toast-messages-for-accessibility-fb610ac364be

These toasts seem not to have the propper aria roles implemented and therefore do not meet the w3c specs.

Closing as invalid.
